src/RA_Dlg_AchEditor.cpp

```diff
@@ -1783,6 +1731,80 @@ INT_PTR Dlg_AchievementEditor::AchievementEditorProc(HWND hDlg, UINT uMsg, WPARA
     return !bHandled;
 }
 
+unsigned int Dlg_AchievementEditor::ParseValue(const std::string& sData, CompVariable::Type nType) const
+{
+    unsigned int nMax = 0xFFFFFFFF;
+
+    int nBase = 16;
+    if (nType == CompVariable::Type::ValueComparison)
+    {
+        auto& pConfiguration = ra::services::ServiceLocator::Get<ra::services::IConfiguration>();
+        if (pConfiguration.IsFeatureEnabled(ra::services::Feature::PreferDecimal))
+            nBase = 10;
+
+        // TODO: limit nMax to size of memory being read
```

Comment:
Why is nMax different with nBase = 10? Isn't the size is base 10 either way?

Reply:
future feature. if the field is 8-bit, nMax should be 0xFF. 0xFFFF for 16-bit, etc.